convert2rhel/logger.py

```diff
@@ -117,6 +116,13 @@ def info(self, msg, *args, **kwargs):
         super(CustomLogger, self).info(msg, *args, **kwargs)
 
 
+class bcolors:
```

Comment:
Nit: I think enum.Enum is exactly for such cases. If compare with your solution I think the difference is that enum is immutable, uses less memory, and has a lot of nice features out of the box (i.e. iterator protocol, nice __str__ and __repr__ ...)

Reply:
Unfortunately, `enum.Enum` is from `python >3.4` and we need to support 2.6. Python future doesn't seem to cover it easily either. Too much work :)

Reply:
:sob: